clientv3/op.go

```diff
@@ -119,16 +119,26 @@ func (op *Op) applyOpts(opts []OpOption) {
 
 type OpOption func(*Op)
 
+// WithLease attaches a lease ID to a key in Put request.
 func WithLease(leaseID lease.LeaseID) OpOption {
 	return func(op *Op) { op.leaseID = leaseID }
 }
+
+// WithLimit limits the number of results to return from Get or Delete request.
 func WithLimit(n int64) OpOption { return func(op *Op) { op.limit = n } }
+
+// WithRev specifies the store revision for Get or Delete request.
 func WithRev(rev int64) OpOption { return func(op *Op) { op.rev = rev } }
+
+// WithSort specifies the order in Get results.
```

Comment:
WithSort specifies the order in Get requests when WithRange or WithPrefix is also specified.